xbstrap/base.py

```diff
@@ -402,6 +402,14 @@ def package_out_dir(self):
 		else:
 			return os.path.join(self.build_root, self._root_yml['directories']['packages'])
 
+	@property
+	def cargo_config_toml(self):
+		yml = self._site_yml.get('general', dict())
+		if 'cargo' in yml:
+			return yml['cargo']['config_toml']
+		else:
+			return None
```

Comment:
This is now defined in `boostrap-site.yml`. I would prefer to specify it in `bootstrap.yml`, as it affects the entire project, not only individual build directories or individual users.